pypeit/specobjs.py

```diff
@@ -134,13 +138,104 @@ def nobj(self):
         """
         return len(self.specobjs)
 
-    def get_std(self):
+    def unpack_object(self, ret_flam=False):
+        """
+        Utility function to unpack the sobjs for one object and
+        return various numpy arrays describing the spectrum and meta
+        data. The user needs to already have trimmed the Specobjs to
+        the relevant indices for the object.
```

Comment:
The method is used in two places, coadd1d and sensfunc.py. It is  for extracing the information that you "actually need" in most cases when dealing with the 1d spectra for fluxing and coadds. If this were used just once I would see your point, but since it is used twice it seems already a justification for the method since I'd have to replicate that code twice.